Unifier - add support for email list (#873)
demisto/demisto-sdk: CHANGELOG.md

# Changelog

> * Added validation for approved content pack usecases and tags.

Reviewer: ```suggestion
* Added validation for approved content pack use-cases and tags.
```

# Changelog
* Added validation for approved content pack usecases and tags.
* Added new code validations for *CommonServerPython* import to `XSOAR-linter`.

> * For partner supported content packs, improved the support email formatting in the integration detailed description.

Reviewer: ```suggestion
* For partner supported content packs, added support for a list of emails.
```

* Added new code validations for *CommonServerPython* import to `XSOAR-linter`.
* Added *default value* and *predefined values* to argument description in **generate-docs** command.
* Added a new validation that checks if *get-mapping-fields* command exists if the intgeration schema has *{ismappable: true}* in **validate** command.
* For partner supported content packs, added support for a list of emails.

> * Added arg_to_array function to sdk-tools.

Reviewer: No need for the last note, it is not relevant for the SDK users.
